src/graphics.rs

```diff
@@ -39,6 +40,7 @@ impl UniformType {
             UniformType::Float2 => 8 * count,
             UniformType::Float3 => 12 * count,
             UniformType::Float4 => 16 * count,
+            UniformType::Int => 4 * count,
```

Comment:
If it is possible - maybe add Int1, Int2, Int3 and Int4 as well?

Reply:
done